platform-stack/register_mqtt_sensor.sh

```diff
@@ -0,0 +1,381 @@
+sudo curl -iX POST \
+  'http://localhost:4041/iot/devices' \
+  -H 'Content-Type: application/json' \
+  -H 'fiware-service: openiot' \
+  -H 'fiware-servicepath: /' \
+  -d '{
+    "devices": [
+      {
+        "device_id":   "traffic001",
+        "entity_name": "urn:ngsi-ld:TrafficFlowObserved:001",
+        "entity_type": "TrafficFlowObserved",
+        "protocol":    "JSON",
+        "transport":   "MQTT",
+        "timezone":    "Europe/Berlin",
+        "attributes": [
+          { "object_id": "car_counter", "name": "count", "type": "Integer" }
+        ]
+      },
+      {
+        "device_id":   "traffic002",
+        "entity_name": "urn:ngsi-ld:TrafficFlowObserved:002",
+        "entity_type": "TrafficFlowObserved",
+        "protocol":    "JSON",
+        "transport":   "MQTT",
+        "timezone":    "Europe/Berlin",
+        "attributes": [
+          { "object_id": "car_counter", "name": "count", "type": "Integer" }
+        ]
+      },
+      {
+        "device_id":   "lights001",
+        "entity_name": "urn:ngsi-ld:Lights:001",
+        "entity_type": "Lights",
+        "protocol":    "JSON",
+        "transport":   "MQTT",
+        "timezone":    "Europe/Berlin",
+        "attributes": [
+          { "object_id": "powerState", "name": "state", "type": "String" }
+        ],
+         "commands": [
+           {
+               "name": "light",
+                "type": "command"
+           }
+         ]
+       },
+       {
+        "device_id":   "lights002",
+        "entity_name": "urn:ngsi-ld:Lights:002",
+        "entity_type": "Lights",
+        "protocol":    "JSON",
+        "transport":   "MQTT",
+        "timezone":    "Europe/Berlin",
+        "attributes": [
+          { "object_id": "powerState", "name": "state", "type": "String" }
+        ],
+         "commands": [
+           {
+               "type": "command",
+               "name": "light"
+           }
+         ]
+       },
+       {
+        "device_id":   "parking001",
+        "entity_name": "urn:ngsi-ld:Parking:001",
+        "entity_type": "Parking",
+        "protocol":    "JSON",
+        "transport":   "MQTT",
+        "timezone":    "Europe/Berlin",
+        "attributes": [
+          { "object_id": "parking_spot_status", "name": "parking_spot_status", "type": "String" }
+        ]
+      },
+      {
+        "device_id":   "ev-charging001",
+        "entity_name": "urn:ngsi-ld:Ev-Charging:001",
+        "entity_type": "Ev-Charging",
+        "protocol":    "JSON",
+        "transport":   "MQTT",
+        "timezone":    "Europe/Berlin",
+        "attributes": [
+          { "object_id": "parking_spot_status", "name": "parking_spot_status", "type": "String" },
+          {"object_id": "capacity", "name": "capacity", "type": "Integer" }
+        ],
+         "commands": [
+           {
+               "name": "ev_charging",
+                "type": "command"
+           }
+         ]
+       },
+       {
+        "device_id":   "ev-charging002",
```

Comment:
why is this separate?

the button is not a new ev charging station, it's the status of charging / not charging of the same station

Reply:
because the button sends his status ("pressed"/"released") on his topic. The charging process will start only if the button is pressed and the ev-charging station receives the command "start charge".  So we need to register the button as well